Earthfile

```
@@ -0,0 +1,247 @@
+os-base:
+    ARG CODE_NAME=focal
+    FROM ubuntu:$CODE_NAME
```

Use the docker image with Ubuntu and `conda`/`mamba` pre-installed, I'm also migrating them to focal (incl. the introduction of version tagged images) over at https://github.com/conda-forge/miniforge-images/pull/2 .
```suggestion
    FROM condaforge/mambaforge:latest
```